src/auth/dto/authcreds.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, IsNotEmpty, IsEmail } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AuthCredsDto {
+  @IsString()
+  @ApiProperty({ description: 'The email of the user' })
+  @IsEmail()
+  @IsNotEmpty()
+  email: string;
+
+  @IsString()
+  @ApiProperty({ description: 'The password of the user' })
+  @IsNotEmpty()
+  password: string;
```

Comment:
PullProofAI

The `AuthCredsDto` class looks well-defined with proper validation decorators for email and password fields. It follows the best practice of using `class-validator` for input validation and `@nestjs/swagger` for API documentation.
This implementation ensures that the email field is a valid email format and not empty, while the password field is not empty.
No changes needed here. Well done!


<!-- This is an auto-generated comment by PullProofAI -->